### ISSUE TITLE
Calendar - Replace Heading with Text

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#6637]
Follow on to: https://github.com/grommet/hpe-design-system/issues/3016
### Expected Behavior
The Calendar's month and year shouldn't be a Heading. This causes the screenreader to interpret it as a heading landmark on the page.

### Actual Behavior
We should use the Text component instead of Heading for the month and year. The month and year container should be a Header component.

### Steps to Reproduce

1. 2. 3.

### Your Environment

- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):
